src/tyr/serializers.cc

```diff
@@ -179,6 +179,34 @@ waypoint(const valhalla::Location& location, bool is_tracepoint, bool is_optimiz
   return waypoint;
 }
 
+/*
+ * This function serializes the via_waypoints object on the leg.
+ */
+valhalla::baldr::json::MapPtr via_waypoint(const valhalla::Location& location,
+                                           const uint64_t geometry_idx) {
+
+  // Create a via waypoint to add to the array
+  auto via_waypoint = json::map({});
+  // Add distance in meters from the input location to the nearest
+  // point on the road used in the route
+  // TODO: since distance was normalized in thor - need to recalculate here
+  //       in the future we shall have store separately from score
```

Comment:
What does this comment mean?

Reply:
Removed comment as this doesn't relate to via_waypoints